go-selinux/label/label_linux.go

```diff
@@ -53,11 +54,11 @@ func InitLabels(options []string) (plabel string, mlabel string, retErr error) {
 				return "", selinux.PrivContainerMountLabel(), nil
 			}
 			if i := strings.Index(opt, ":"); i == -1 {
-				return "", "", errors.Errorf("Bad label option %q, valid options 'disable' or \n'user, role, level, type, filetype' followed by ':' and a value", opt)
+				return "", "", fmt.Errorf("Bad label option %q, valid options 'disable' or \n'user, role, level, type, filetype' followed by ':' and a value", opt)
```

Comment:
I hate it that they didn't add a `errors.Errorf()` 😞

Reply:
Well, could be as easy as `var Errorf = fmt.Errorf` but it's impossible as `fmt` imports `errors` and so `errors` can't import `fmt`.